extensions/optaplanner/runtime/src/main/java/io/quarkus/optaplanner/nativeimage/KogitoSubstitutions.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.optaplanner.nativeimage;
+
+import java.lang.reflect.Method;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.RecomputeFieldValue;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@TargetClass(className = "org.drools.core.rule.builder.dialect.asm.ClassGenerator")
+final class Target_org_drools_core_rule_builder_dialect_asm_ClassGenerator {
```

Comment:
Why is the class name different from `KogitoSubstitutions`?

Reply:
It doesn't really matter since it's going away really really soon

Reply:
Well yeah, it's just that it's confusing to look at this class and see another name :)

Reply:
The idea was that multiple substitutions might have been needed. For example like Netty.
It turned out that it wasn't needed 😊